src/components/Onfido/BaseOnfidoWeb.js

```diff
@@ -105,7 +105,7 @@ class Onfido extends React.Component {
                 Log.hmmm('Onfido user closed the modal');
             },
             language: {
-                locale: this.props.preferredLocale,
+                locale: this.props.preferredLocale === CONST.LOCALES.ES ? CONST.ONFIDO.ES_ES_LOCALE : CONST.LOCALES.EN,
```

Comment:
Update this with the new place, and let's use `this.props.preferredLocale` for the fallback.